### PR TITLE
pkp/pkp-lib#1403  add redirectUrl to loginAs in Participants list of …

### PR DESCRIPTION
**Pull request by forgive38**
…a submission in order to redirect user to the same submission when loginAs/logoutAs

### CODE REVIEW

File: controllers/grid/users/stageParticipant/StageParticipantGridRow.inc.php [hunk 1]
```diff
@@ -84,13 +85,36 @@ function initialize($request, $template = null) {
...
+				} else {
+					$redirectUrl = $dispatcher->url(
+						$request,
+						ROUTE_PAGE,
+						$context->getPath(),
+						'workflow',
+						'access',
+						$submission->getId()
+					);
+				}
```
Comment (defstat) [inline controllers/grid/users/stageParticipant/StageParticipantGridRow.inc.php]: @forgive38 Here we could have the `$redirectUrl` be assigned bellow the `if` statement, with only the `'authorDashboard'` or `'workflow'` and `'submission'` or `'access'` assigned to two separate variables, and then passed as parameters to the `->url(..)` function.

**Comment by forgive38** [comment]
@defstat : is it ok for you ?

**Comment by forgive38** [comment]
@defstat  : this one pkp/ojs#2462 (tests only)

**Comment by defstat** [comment]
Yes like that commit on ojs. When you change something on pkp-lib, you can force push the commit again on your ojs branch in order for the tests to run again, taking into account the newly pushed changes. If you need help on that, I will be happy to give more instructions

**Comment by forgive38** [comment]
sorry, but I  don't understand :(

for me, this commit pkp/ojs#2462 is the commit in ojs with submodule update references my branch of pkp-lib and tests passed : https://travis-ci.org/pkp/ojs/builds/584191712?utm_source=github_status&utm_medium=notification

what can I do more ?
thank you for you help.

**Comment by defstat** [comment]
@forgive38 nothing more thanks. 

@asmecher the code could be merged. Thanks

**Pull request merged**